### ISSUE TITLE
Customized Voice Conversion

### ISSUE THREAD

**Issue by zeng-yifei** [miaomiaomiao-LJY/ppg_tacotron#9]
After training enough epochs, the result of imitating cmu arctic finally becomes more and more satisfying!

But when it comes to create a customized voice converter, I have no idea about what wav feature I should preserve when making my own "cmu arctic" dataset. Do I need to keep the every feature **absolute same** with cmu's dataset, like 2 second of wav duration?

Plus, I saw the issue of applying this project on mandarin. If using pyPinYin, do I need to make bigger dataset than TIMIT due to the great variance of pinyin? And what lib should I use to get every word's duration to make a phn file like TIMIT? ( which is the most confusing question for me. )

**Comment by LJY-M** [comment]
> After training enough epochs, the result of imitating cmu arctic finally becomes more and more satisfying!
> 
> But when it comes to create a customized voice converter, I have no idea about what wav feature I should preserve when making my own "cmu arctic" dataset. Do I need to keep the every feature **absolute same** with cmu's dataset, like 2 second of wav duration?
> 
> Plus, I saw the issue of applying this project on mandarin. If using pyPinYin, do I need to make bigger dataset than TIMIT due to the great variance of pinyin? And what lib should I use to get every word's duration to make a phn file like TIMIT? ( which is the most confusing question for me. )

Congratulations!

Similar to the "CMU Arctic" dataset, you only need to record enough of your own audio (over 30 minutes) to achieve custom speech conversion.

I would argue that Mandarin pinyin is richer and more complex than English phonemes, so a larger dataset than the TIMIT dataset is needed.

Finally, You can use  **Montreal-Force-Aligner** to make phn files on new datasets.

**Comment by zeng-yifei** [comment]
Thanks for your concrete advice and really really nice work!
I will follow your steps and continue trying to obtain a satisfying result on a customized dataset!

**Comment by WildStrom** [comment]
> Thanks for your concrete advice and really really nice work! I will follow your steps and continue trying to obtain a satisfying result on a customized dataset!

I also don't know how to customize the voice conversion. Do you get new progerss？

**Comment by WildStrom** [comment]
> > After training enough epochs, the result of imitating cmu arctic finally becomes more and more satisfying!
> > But when it comes to create a customized voice converter, I have no idea about what wav feature I should preserve when making my own "cmu arctic" dataset. Do I need to keep the every feature **absolute same** with cmu's dataset, like 2 second of wav duration?
> > Plus, I saw the issue of applying this project on mandarin. If using pyPinYin, do I need to make bigger dataset than TIMIT due to the great variance of pinyin? And what lib should I use to get every word's duration to make a phn file like TIMIT? ( which is the most confusing question for me. )
> 
> Congratulations!
> 
> Similar to the "CMU Arctic" dataset, you only need to record enough of your own audio (over 30 minutes) to achieve custom speech conversion.
> 
> I would argue that Mandarin pinyin is richer and more complex than English phonemes, so a larger dataset than the TIMIT dataset is needed.
> 
> Finally, You can use **Montreal-Force-Aligner** to make phn files on new datasets.

so as I understand， the TIMIT dataset is not enough to change pingyin in Chinese，is that right？

**Comment by LJY-M** [comment]
>

> > > After training enough epochs, the result of imitating cmu arctic finally becomes more and more satisfying!
> > > But when it comes to create a customized voice converter, I have no idea about what wav feature I should preserve when making my own "cmu arctic" dataset. Do I need to keep the every feature **absolute same** with cmu's dataset, like 2 second of wav duration?
> > > Plus, I saw the issue of applying this project on mandarin. If using pyPinYin, do I need to make bigger dataset than TIMIT due to the great variance of pinyin? And what lib should I use to get every word's duration to make a phn file like TIMIT? ( which is the most confusing question for me. )
> > 
> > 
> > Congratulations!
> > Similar to the "CMU Arctic" dataset, you only need to record enough of your own audio (over 30 minutes) to achieve custom speech conversion.
> > I would argue that Mandarin pinyin is richer and more complex than English phonemes, so a larger dataset than the TIMIT dataset is needed.
> > Finally, You can use **Montreal-Force-Aligner** to make phn files on new datasets.
> 
> so as I understand， the TIMIT dataset is not enough to change pingyin in Chinese，is that right？

Yes, the TIMIT data set is too small to complete Chinese phoneme recognition.
